js/ui/appDisplay.js

```diff
@@ -1089,6 +1116,13 @@ const AppSearchProvider = new Lang.Class({
     createResultObject: function (resultMeta) {
         let app = this._appSys.lookup_app(resultMeta['id']);
         return new AppIcon(app);
+    },
+
+    activateResult: function(appId) {
```

Comment:
I took that from the older codebase, where there was no AppIcon being created. I'll remove this method